buildconfig/Jenkins/buildscript

```diff
@@ -58,7 +58,9 @@ terminate_xvfb_sessions
 # System discovery
 ###############################################################################
 USE_CORE_DUMPS=true
-if [[ ${NODE_LABELS} == *rhel7* ]] || [[ ${NODE_LABELS} == *centos7* ]] || [[ ${NODE_LABELS} == *scilin7* ]]; then
+if [[ ${JOB_NAME} == *master_conda* ]]; then
```

Comment:
How about just `*conda*` so if we want to test other branches we can just do this based off the job name still?

Reply:
This causes a bug actually with a change I made to their name I will adjust the script.